### PR TITLE
Allow batch size > 1

### PR DESCRIPTION
**Pull request by vsuryamurthy**
Allows training with batch size > 1. Tested with batch sizes 2 and 3.

### CODE REVIEW

File: keras_maskrcnn/losses.py [hunk 1]
```diff
@@ -28,60 +28,91 @@ def _mask(y_true, y_pred, iou_threshold=0.5, mask_size=(28, 28)):
...
+            parallel_iterations=parallel_iterations
+        )
 
-        return mask_loss
+        return keras.backend.mean(mask_batch_loss)
 
     return _mask_conditional
+
+
+def compute_loss(
```
Comment (hgaiser) [inline keras_maskrcnn/losses.py]: Since this is a toplevel function, I suggest to rename it to `compute_mask_loss`. I had half a mind to move this function inside the scope of `mask`..

**Comment by hgaiser** [comment]
LGTM! One comment on the function name.

**Comment by vsuryamurthy** [comment]
The change to the function name has been made

**Comment by hgaiser** [comment]
Feel free to merge once the tests have passed.
